### PR TITLE
fixed issue_#26 and issue_#27

### PR DESCRIPTION
**Pull request by developer0623**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,13 +30,15 @@
     "https": "^1.0.0",
     "libsodium-wrappers": "^0.5.4",
     "libsodium-wrappers-sumo": "^0.5.4",
-    "tsconfig-paths-webpack-plugin": "^3.2.0"
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
```
Comment (vishakh) [inline package.json]: Out of curiosity, what is the purpose of this package?

File: src/utils/CryptoUtils.ts [hunk 2]
```diff
@@ -25,6 +26,10 @@ export function generateSaltForPwHash() {
  * @returns {Buffer}    Concatenated bytes of nonce and cipher text
  */
 export function encryptMessage(message: string, passphrase: string, salt: Buffer) {
+    const passwordLength = getPasswordStrength(passphrase);
```
Comment (vishakh) [inline src/utils/CryptoUtils.ts]: Please rename `passwordLength` to `passwordStrength`.

File: src/utils/CryptoUtils.ts [hunk 3]
```diff
@@ -143,4 +148,13 @@ export function getKeysFromMnemonicAndPassphrase(mnemonic: string, passphrase: s
  */
 export function generateMnemonic(): string {
     return bip39.generateMnemonic(160)
+}
+
+/**
+ * Checking the password strength using zxcvbn
+ * @returns {number}    Fifteen word mnemonic
```
Comment (vishakh) [inline src/utils/CryptoUtils.ts]: Please change "Fifteen word mnemonic" to "Password score"

File: test/CryptoUtils.spec.ts [hunk 4]
```diff
@@ -35,9 +35,31 @@ describe('encryptMessage() and decryptMessage()', () => {
...
     });
+    it('should be not encrypt if the password strength is less than 3', () => {
+        const salt = cryptoUtils.generateSaltForPwHash();
+        const message = "hello";
+        const passphrase = "password";
+        try {
+            const encrypted = cryptoUtils.encryptMessage(message, passphrase, salt);
+            const decrypted = cryptoUtils.decryptMessage(encrypted, passphrase, salt);
+        } catch (err) {
+            expect(err.messa).to.equal('The password strength should not be less than 3.');
```
Comment (vishakh) [inline test/CryptoUtils.spec.ts]: Should `messa` be `message`?

File: package.json (hunk 1 above)
Comment (developer0623) [reply package.json]: https://www.npmjs.com/package/tsconfig-paths-webpack-plugin

Comment (developer0623) [reply package.json]: Use this to load modules whose location is specified in the paths section of tsconfig.json when using webpack. This package provides the functionality of the tsconfig-paths package but as a webpack plug-in.

**Comment by vishakh** [comment]
Addressed #26 and #27

**Pull request merged**